### ISSUE TITLE
Preferences broken in 0.7 (invalid gconf 'window_sticky' key)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [yendo/gphotoframe#49]
```
What steps will reproduce the problem?
1. Run version 0.7
2. Go to preferences window
3. Traceback (most recent call last):
  File "/usr/lib/python2.6/site-packages/gphotoframe/preferences.py", line
32, in start
    checkbutton1.set_active(sticky)
TypeError: an integer is required

What is the expected output? What do you see instead?
Should see the preferences window but this is not shown.

What version of the product are you using? On what operating system?
Version 0.7 on Fedora 12, running either the RPM or from tar file.

```

Original issue reported on code.google.com by `graham_a...@hotmail.com` on 6 May 2010 at 7:37

**Comment by GoogleCodeExporter** [comment]
```
Please try the following command.

sudo gconf-schemas --register gphotoframe.schemas
```

Original comment by `yendo0206` on 6 May 2010 at 10:22

**Comment by GoogleCodeExporter** [comment]
```
Seems to make no difference.  If I install or uninstall the schema, I still get 
the 
error shown.

I tried the following:
cd ~/Download/gphotoframe-0.7
export GCONF_CONFIG_SOURCE=`gconftool-2 --get-default-source`
gconftool-2 --makefile-uninstall-rule ./share/gphotoframe.schemas.in
./gphotoframe
[ right click, choose preferences ]
Traceback (most recent call last):
  File "/home/gwhite/Download/gphotoframe-0.7/lib/preferences.py", line 32, in start
    checkbutton1.set_active(sticky)
TypeError: an integer is required
```

Original comment by `graham_a...@hotmail.com` on 6 May 2010 at 11:26

**Comment by GoogleCodeExporter** [comment]
```
> sudo gconf-schemas --register gphotoframe.schemas

This work must be necessary. 
It takes time to reflect the registering.

For the time being you can evade this error by the following command.

gconftool-2 --type bool --set /apps/gphotoframe/window_sticky 0
```

Original comment by `yendo0206` on 6 May 2010 at 11:52

**Comment by GoogleCodeExporter** [comment]
```
Yes I did try that too but it yields another error which is where I stopped 
looking...

This time I tried the following...
gconftool-2 --type bool --set /apps/gphotoframe/window_sticky 0
./gphotoframe
[ right click, choose preferences ]
Warning: the desktop entry of gphotoframe is not found.
Traceback (most recent call last):
  File "/home/gwhite/Download/gphotoframe-0.7/lib/preferences.py", line 42, in start
    gui, "treeview2", plugins.PluginListStore(), self.prefs)
  File "/home/gwhite/Download/gphotoframe-0.7/lib/plugins/__init__.py", line 47, in 
__init__
    list = [ obj.is_available(), obj.get_icon_pixbuf(), name ]
  File "/home/gwhite/Download/gphotoframe-0.7/lib/plugins/base.py", line 24, in 
get_icon_pixbuf
    pixbuf = self.icon().get_pixbuf()
  File "/home/gwhite/Download/gphotoframe-0.7/lib/utils/iconimage.py", line 27, in 
get_pixbuf
    pixbuf = gtk.gdk.pixbuf_new_from_file(file)
TypeError: pixbuf_new_from_file() argument 1 must be string, not None
```

Original comment by `graham_a...@hotmail.com` on 6 May 2010 at 12:46

**Comment by GoogleCodeExporter** [comment]
```
This is another problem. I opened new issue #50.

```

Original comment by `yendo0206` on 6 May 2010 at 2:03

**Comment by GoogleCodeExporter** [comment]
Original comment by `yendo0206` on 6 May 2010 at 2:21

**Comment by GoogleCodeExporter** [comment]
```
Fixed in revision e480ebf43d.
```

Original comment by `yendo0206` on 7 May 2010 at 3:32
- Changed title: **Preferences broken in 0.7 (invalid gconf 'window_sticky' key)**
- Changed state: **Fixed**
